src/database/attendee-db.ts

```diff
@@ -0,0 +1,52 @@
+import { getModelForClass, prop } from "@typegoose/typegoose";
+import { Databases, generateConfig } from "database.js";
+import mongoose from "mongoose";
+
+enum AttendeeDB {
+    METADATA = "metadata",
+    PROFILE = "profile",
+}
+
+export class AttendeeMetadata {
+    @prop({ required: true })
+    public _id: string;
+
+    @prop({ required: true })
+    public userId: string;
+
+    @prop({ required: true })
+    public foodWave: number;
+
+    @prop({ required: true })
+    public score: number;
```

Comment:
points?

Reply:
if this is points - it should be part of profile, not metadata

Reply:
p-oh-tato poh-ta-to. any preferences/

Reply:
i thought metadata was data that we change, and profile data is data that the user changes

Reply:
yes, points.